test/system/lifecycle/lifecycle_test.go

```diff
@@ -276,7 +277,8 @@ func requestAPIServer(ctx context.Context, caBundle []byte, apiserverURL, bearer
 	client := &http.Client{
 		Transport: &http.Transport{
 			TLSClientConfig: &tls.Config{
-				RootCAs: caCertPool,
+				RootCAs:    caCertPool,
+				MinVersion: tls.VersionTLS12,
```

Comment:
Just wondering if this was reported as a finding? Clients should default to 1.2 since go 1.18 https://go.dev/doc/go1.18#tls10

Reply:
Yes, it did.

```bash
$ make check
> Check
Executing golangci-lint
...
test/system/lifecycle/lifecycle_test.go:279:22: G402: TLS MinVersion too low. (gosec)
                        TLSClientConfig: &tls.Config{
                                RootCAs: caCertPool,
                        },
...
```